release notes/v0.53.0.md

```diff
@@ -0,0 +1,173 @@
+
+That warning is now an error.
+
+In the future we will move the default value to `blob` and remove the error.
+
+## New features
+
+The new features include:
+- Native ESM support, which also brings some quality of life JavaScript features
+- Blob support in the experimental websockets module.
+- Experimental OpenTelemetry metrics output.
+- Consolidating cloud related commands and features under `k6 cloud`
+
+### Native ESM support [#3456](https://github.com/grafana/k6/pull/3456)
+
+TODO: expand, talk about additional JavaScript features now support, top-level-await, dynamic import, compatibility-mode history
+
+### Blob support in the experimental websockets module [grafana/xk6-websockets#74](https://github.com/grafana/xk6-websockets/pull/74)
+
+// TODO
+
+### Experimental OpenTelemetry Output [#3834](https://github.com/grafana/k6/pull/3834)
+
+This release introduces a new experimental output for OpenTelemetry. This allows users to send k6 metrics to any OpenTelemetry-compatible backends. More details and usage examples could be found in the [documentation](https://grafana.com/docs/k6/latest/results-output/real-time/opentelemetry/).
+
+To output metrics to OpenTelemetry, use the `experimental-opentelemetry` output option:
+
+```bash
+ k6 run -o experimental-opentelemetry examples/script.js
+```
+
+If you have any feedback or issues, please let us know directly in [the extension repository](https://github.com/grafana/xk6-output-opentelemetry/issues).
+
+### Consolidating cloud features under `k6 cloud` [#3813](https://github.com/grafana/k6/pull/3813)
+
+// TODO
+
+## UX improvements and enhancements
+
+- [#3783](https://github.com/grafana/k6/pull/3783) Set correct exit code on invalid configurations. Thank you @ariasmn!
+## Bug fixes
+
+- [#3845](https://github.com/grafana/k6/pull/3845) Fix logging to file sometimes missing lines. Thank you @roobre!
+
+## Maintenance and internal improvements
+
+- [#3792](https://github.com/grafana/k6/pull/3792), [#3817](https://github.com/grafana/k6/pull/3817), [#3863](https://github.com/grafana/k6/pull/3863)  Finalize moving to Sobek, our fork of goja.
+- [#3815](https://github.com/grafana/k6/pull/3815), [#3840](https://github.com/grafana/k6/pull/3840), [#3821](https://github.com/grafana/k6/pull/3821), [#3836](https://github.com/grafana/k6/pull/3836), [#3840](https://github.com/grafana/k6/pull/3840), [#3844](https://github.com/grafana/k6/pull/3844), [#3821](https://github.com/grafana/k6/pull/3821)  Update dependencies.
+- [#3830](https://github.com/grafana/k6/pull/3830), [#3831](https://github.com/grafana/k6/pull/3831), [#3862](https://github.com/grafana/k6/pull/3862) Refactoring and cleanup around ESM PR.
+
+
+## _Optional_ Roadmap
+// TODO: top-level-await, dynamic import, import.meta.resolve, `global` and compatibility-mode?!?
```

Comment:
Anyone wants anything else here?

Shall we remind people what will break in v0.54.0? Maybe have a list of breaking changes for upcoming releases ?

Reply:
Yeah, having a list of breaking changes would be a good idea.

Reply:
I think it's fair to announce here another iteration of the `k6 cloud` (sub)commands, with things like https://github.com/grafana/k6/pull/3818 and https://github.com/grafana/k6/pull/3835.

Wdyt? @oleiade